superset/viz.py

```diff
@@ -484,8 +485,8 @@ def get_df_payload(self, query_obj=None, **kwargs):
             "form_data": self.form_data,
             "is_cached": self._any_cache_key is not None,
             "query": self.query,
-            "from_dttm": self.from_dttm,
-            "to_dttm": self.to_dttm,
+            "from_dttm": self.from_dttm if hasattr(self, "from_dttm") else None,
```

Comment:
NIT: I think it may be better to init `self.from_dttm = None` in the constructor of `BaseViz`, initializing all the attrs that may get assigned in the lifecycle of the object there

Reply:
Nit accepted. I need to close + reopen this PR again anyway because Travis won't play nice.